tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -113,25 +113,70 @@ def test_slurm(
         partition="gpu",
         instance_type=gpu_instance_type,
         max_count=5,
-        gpu_per_instance=_get_num_gpus_on_instance(gpu_instance_type_info),
-        gpu_type="m60",
+        gpu_instance_type_info=gpu_instance_type_info,
     )
     # Test torque command wrapper
     _test_torque_job_submit(remote_command_executor, test_datadir)
     assert_no_errors_in_logs(remote_command_executor, "slurm")
-    # Test compute node bootstrap timeout
-    if scheduler == "slurm":  # TODO enable this once bootstrap_timeout feature is implemented in slurm plugin
-        _test_compute_node_bootstrap_timeout(
-            cluster,
-            pcluster_config_reader,
-            remote_command_executor,
-            compute_node_bootstrap_timeout,
-            scaledown_idletime,
-            gpu_instance_type,
-            clustermgtd_conf_path,
-            slurm_root_path,
-            slurm_commands,
-        )
+    _test_compute_node_bootstrap_timeout(
+        cluster,
+        pcluster_config_reader,
+        remote_command_executor,
+        compute_node_bootstrap_timeout,
+        scaledown_idletime,
+        gpu_instance_type,
+        clustermgtd_conf_path,
+        slurm_root_path,
+        slurm_commands,
+    )
+
+
+@pytest.mark.usefixtures("instance", "os")
+def test_slurm_bug_17399(
+    region, pcluster_config_reader, clusters_factory, test_datadir, architecture, scheduler_commands_factory
+):
+    """
+    Test Slurm bug 17399
+
+    Check if certain valid combinations of sbatch options lead to a submission error
```

Comment:
Please add some extra details on what this code will actually test.

While `bug 17399` is really specific ...it is hard to decode for anyone that wasn't involved in the investigation ...and I doubt we will remember the specific case in few months.

Reply:
We discussed offline and decided just to rename the test.